Lib/csv.py

```diff
@@ -193,6 +193,9 @@ def __next__(self):
                 d[key] = self.restval
         return d
 
+    def __repr__(self):
+        return "%s('%s')" % (self.__class__.__name__, self.dialect)
```

Comment:
Could we use an f-string instead? I don't see the need to use the ancient `%` operator:

```suggestion
        return f"{self.__class__.__name__}(self.dialect!r})"
```

(This goes for all the other cases as well.)

Reply:
I'm also not sure whether we should a fully-qualified name or not (namely, should we include the module's name or not?)